app/models/work.rb

```diff
@@ -617,7 +621,8 @@ def series_attributes=(attributes)
         new_series.creatorships.build(pseud: pseud)
       end
       new_series.save
```

Comment:
Can we remove this line, following @tickinginstant's comment [in the other thread](https://github.com/otwcode/otwarchive/pull/3634#discussion_r553664121), to avoid creating the new series if we can't save the serial work?